app/Http/Controllers/Staff/UploaderController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * NOTICE OF LICENSE.
+ *
+ * UNIT3D Community Edition is open-sourced software licensed under the GNU Affero General Public License v3.0
+ * The details is bundled with this project in the file LICENSE.txt.
+ *
+ * @project    UNIT3D Community Edition
+ *
+ * @author     HDVinnie <hdinnovations@protonmail.com>
+ * @license    https://www.gnu.org/licenses/agpl-3.0.en.html/ GNU Affero General Public License v3.0
+ */
+
+namespace App\Http\Controllers\Staff;
+
+use App\Http\Controllers\Controller;
+use App\Models\Group;
+use App\Models\User;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\DB;
+
+/**
+ * @see \Tests\Todo\Feature\Http\Controllers\Staff\BanControllerTest
+ */
+class UploaderController extends Controller
+{
+    /**
+     * Display Stats.
+     */
+    public function index(): \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+    {
+        $uploaderGroup = cache()->rememberForever('uploader_group', fn () => Group::where('slug', '=', 'uploader')->pluck('id'));
```

Comment:
This is troublesome. This is yet another "keyword" that has to be blacklisted from everywhere. You will need to add a migration to set the `system_required` flag to true for this group. Or maybe add an `is_uploader` column to the `groups` table. What do you think @HDVinnie? I would probably recommend the `is_uploader` column for better database structure.

Reply:
Fixed. Added the `is_uploader` flag.